sound/soc/sof/loader.c

```diff
@@ -113,8 +113,8 @@ EXPORT_SYMBOL(snd_sof_load_firmware_memcpy);
 int snd_sof_run_firmware(struct snd_sof_dev *sdev)
 {
 	struct sof_dsp_power_state state = {
-                .state = SOF_DSP_PM_D0,
-        };
+		.state = SOF_DSP_PM_D0,
+	};
```

Comment:
this should be the first patch.

Reply:
Uh, sorry, I put this last on purpose. Now fixed in updated series.